tests/app/test_queue.py

```diff
@@ -16,15 +16,13 @@ def pmr_redis_config():
 
 redis = create_redis_fixture(scope="function")
 REDIS_ELEMENTS_COUNT = 123
-QNAME_SUFFIX = "test"
```

Comment:
Can we keep this constant but instantiate it to app.config["NOTIFY_ENVIRONMENT"]) instead? That would be easier to read in the test code.

Reply:
Sorry I dont agree.. `QNAME_SUFFIX ` is surplus to requirements.. hence not DRY

Reply:
the idea of a suffic appear broad, and put in place of the `environment` it seems

Reply:
If we are to go with DAMP then `suffix` appears vague because its isn't descriptive because the sections of the name appear to follow this pattern `queue:environment:id`

Reply:
Ok I think I see the misunderstanding. The suffix does not follow `queue:environment:id` pattern you described. It does not make sense anyway with the examples I provided in my last reply. The suffix can be anything deemed good and descriptive for the usage of the queue, i.e. `sms` or `email`. 

This derives from a requirement that Jumana explained earlier this week in a call where we were all present where we need different queues based on the notification type... but that will expand in the future on the notification priority as well.

Reply:
Also, at that point, we are talking about the parameter name `suffix` which is out of scope of the initial talk around the usage of a constant in a test. So I'll consider this closed for now. We can change the parameter name later on if we want to in a separate PR, but for now, I will merge this one based on the recent changes I brought to it. We need to move forward and the scope for this PR were:

* have a way to customize inflight name
* write tests around it and avoid repetition of the double `:`